tests/integration/build_invoke/node/test_node_16.py

```diff
@@ -37,6 +37,10 @@ class BuildInvoke_nodejs16_x_cookiecutter_quick_start_sqs(BuildInvokeBase.BuildI
     directory = "nodejs16.x/sqs"
 
 
+class BuildInvoke_nodejs16_x_cookiecutter_quick_start_response_streaming(BuildInvokeBase.QuickStartWebBuildInvokeBase):
```

Comment:
Why is node16 based on `BuildInvokeBase.QuickStartWebBuildInvokeBase` but node18 is `BuildInvokeBase.BuildInvokeBase`? I would think they would be based on the same thing, unless I am missing a difference is the two versions.

Reply:
Changed.